File: ASMR.Web/ReleaseInformation/Common/WebReleaseInformation.cs
//
// asmr: Coffee Beans Management Solution
// © 2021 Pandora Karya Digital. All right reserved.
//
// Written by Danang Galuh Tegar Prasetyo [connect@danang.id]
//
// WebReleaseInformation.cs
//

using System.Text.Json.Serialization;

namespace ASMR.Web.ReleaseInformation.Common;

public class WebReleaseInformation : GenericReleaseInformation
{
	[JsonPropertyName("Status")] public StoreReleaseInformation Status { get; set; }
}

File: ASMR.Web/ClientApp/src/core/release/common/WebReleaseInformation.ts
import GenericReleaseInformation from "asmr/core/release/common/GenericReleaseInformation"
import StoreReleaseInformation from "asmr/core/release/common/StoreReleaseInformation"

interface WebReleaseInformation extends GenericReleaseInformation {
	Status: StoreReleaseInformation
}

export default WebReleaseInformation
